src/pages/CollectionsPage.tsx

```diff
@@ -1,16 +1,34 @@
-import { Fragment } from "react"
-import HeaderBar from "../components/HeaderBar/HeaderBar"
+import { Fragment, useEffect, useState } from "react"
 import { Container, Row, Col } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../components/HeaderBar/HeaderBar"
+import GetYourCollections from "../api/collections-api";
 
 const CollectionsPage = () => {
+    const [result, setResult] = useState<[] | Array<Collection>>([])
+
+    // inspired by this Stackflow answer:https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret
+    useEffect(() => {
+        async function fetchCollection() {
+            let response = await GetYourCollections()
+            setResult(response.result)
+        }
+        fetchCollection()
+    }, [])
+
     return (
         <Fragment>
```

Comment:
FYI there is a short-syntax for Fragments :)
https://reactjs.org/docs/fragments.html#short-syntax